utils/type.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
             '\\/(?:(ses-[a-zA-Z0-9]+)' +
             '\\/)?meg' +
             '\\/\\1(_\\2)?(?:_task-[a-zA-Z0-9]+)?(?:_acq-[a-zA-Z0-9]+)?(?:_run-[0-9]+)?(?:_proc-[a-zA-Z0-9]+)?(?:_part-[0-9]+)?' +
-            '(_meg.(ctf|fif|4d|kit|kdf|itab)|(_meg.ds\\/.*)|(_events.tsv|_channels.tsv|_meg.json|_fid.json|_photo.jpg|_headshape.pos))$');
+            '(_meg.(ctf|fif|4d|kit|kdf|itab)|(_meg.ds\\/.*)|(_events.tsv|_channels.tsv|_meg.json|_fid.json|_fidphoto.jpg|_headshape.pos|.shape|.shape_info))$');
```

Comment:
umm ... I not sure I follow. In [BEP008 specification](https://docs.google.com/document/d/1FWex_kSPWVh_f4rKgd5rxJmxlboAPtQlmBc1gyZlRZM/edit#heading=h.d3fiyf6uw806), it says it should be `_photo.jpg` not `fidphoto.jpg`

Reply:
this seems to have changed in the spec during the implementation of the examples... I will investigate.